doc/gocc_user_guide.tex

```diff
@@ -750,25 +776,15 @@ \section{Example: Using an AST} \label{sec:example ast}
 
 	\begin{verbatim}
 		> go test -v .
-		warning: building out-of-date packages:
-		        github.com/goccmack/gocc/example/astx/token
-		        github.com/goccmack/gocc/example/astx/ast
-		        github.com/goccmack/gocc/example/astx/errors
-		        github.com/goccmack/gocc/example/astx/parser
-		        github.com/goccmack/gocc/example/astx/lexer
-		installing these packages with 'go test -i .' will speed future tests.
-
-		=== RUN TestPass
+		=== RUN   TestPass
 		input: a b c d e f
 		output: [a b c d e f]
-		--- PASS: TestPass (0.00 seconds)
-		=== RUN TestFail
+		--- PASS: TestPass (0.00s)
+		=== RUN   TestFail
 		input: a b ; d e f
-		--- FAIL: TestFail (0.00 seconds)
-		ast_test.go:23: 	Error: illegal -1(-1) ; @ 1:5, expected one of: id $
-		        FAIL
-		exit status 1
-		FAIL	github.com/goccmack/gocc/example/astx	0.015s	\end{verbatim}
+		--- PASS: TestFail (0.00s)
```

Comment:
Ok great.  Can we make that test do a t.Log with the error and then reflect that output in the docs.  Maybe that will help the docs make more sense, what do you think?